pytorch_lightning/utilities/cli.py

```diff
@@ -76,7 +76,7 @@ def __init__(
         self,
         parser: LightningArgumentParser,
         config: Union[Namespace, Dict[str, Any]],
-        config_filename: str = 'config.yaml'
+        config_filename: str,
```

Comment:
Why do you need to remove the default here?

Reply:
That default will never be used because in the only place where `﻿﻿﻿﻿SaveConfigCallback` should be instantiated its value is given.